src/chainparams.h

```diff
@@ -14,6 +14,48 @@
 
 #include <vector>
 
+/** Default for -minrelaytxfee, minimum relay fee for transactions */
+static const unsigned int DEFAULT_MIN_RELAY_TX_FEE = 1000;
+//! -maxtxfee default
```

Comment:
nit: i agree with getting these values out of main.h but isn't mempool.h a better place?

Reply:
I like the idea of "quasi-consensus" -- its not a consensus parameter, but a parameter that causes negative network-wide consequences if its not consistent.  If you think about these values from that perspective I think become quasi-consensus chain parameters...